css/layout/footer.css

```
@@ -1,254 +1,254 @@
+  cursor: pointer;
+  color: #31302e;
 
-	margin-top: 0.4rem;
-	line-height: 1.35rem;
-	text-align: left;
-	word-wrap: break-word;
-	text-decoration: none;
+  margin-top: 0.4rem;
+  line-height: 1.35rem;
+  text-align: left;
+  word-wrap: break-word;
```

Stylelint (17.2.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)